capture/engine_scripts/puppet/loadLocalStorage.js

```diff
@@ -0,0 +1,40 @@
+var fs = require('fs');
+  
+/*
+  This code is called after the command to navigate has been fired. 
+  before the delay, before the wait for an element to render (readySelector) and before the code that runs just before taking the picture (readyEvent).
+
+  If the data is like this:
+  +------------------------+--------------+
+  |         Key:           |     value:   |
+  +------------------------+--------------+
+  |amplitude_lastEventId   |             2|
+  |amplitude_lastEventTime | 1543336128855|
+  |amplitude_sessionId     | 1543336128827|
+  |amplitude_unsent        |            []|
+  +------------------------+--------------+
+
+  The json should be like this:
+  {
+    "amplitude_lastEventId": "2",
+    "amplitude_lastEventTime": "1543336128855",
+    "amplitude_sessionId": "1543336128827",
+    "amplitude_unsent": "[]"
+  }
+*/
+
+async function loadLocalStorageData(path, page){

```

Comment:
Please model this after the `capture/engine_scripts/puppet/loadCookies.js` file.   Guard against bad file paths etc.   

Please implement as an add in -- like `loadCookies` -- see: `capture/engine_scripts/puppet/onBefore.js`

Do not implement this on the main codepath.